instrumentation-api-incubator/src/test/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlStatementSanitizerTest.java

```diff
@@ -58,6 +58,36 @@ void veryLongSelectStatementsAreOk() {
     assertThat(result).isEqualTo(expected);
   }
 
+  @Test
+  void changeSchemaStatementsAreOk() {
+    String createTableStatement = "CREATE TABLE `table`";
```

Comment:
a paramaterized test would be easier to read

Reply:
Done.